worker/contributors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Instructions: Once you make a contribution to the repo, add yourself to this map,
+ * where the key is your user id in stacker.news prod,
+ * and the value map contains your GitHub username, and your SN nym.
+ * Right now, the GitHub usernames and SN nyms are not used, but maybe we'll use them
+ * in the future for a fancier UI decoration.
+ */
+const contributorMap = {
+  616: {
+    github: 'huumn',
+    sn: 'k00b'
+  },
+  946: {
+    github: 'kerooke',
+    sn: 'kr'
+  },
+  6030: {
```

Comment:
how do you know my user ID? have you received a database dump? lol

wanted to mention that contributors not necessarily know their user IDs on SN

Reply:
I tried that too but I thought it's not possible to see the data because of SSR.

But I'll look more closely now, lol